test/SPV_INTEL_media_block_io.ll

```diff
@@ -11,18 +11,22 @@
 ;}
 
 ; RUN: llvm-as < %s -o %t.bc
-; RUN: llvm-spirv %t.bc -spirv-text -o - | FileCheck %s
+; RUN: llvm-spirv %t.bc -spirv-text -o %t.txt
+; RUN: FileCheck < %t.txt %s --check-prefix=CHECK-SPIRV
+; RUN: llvm-spirv %t.bc -o %t.spv
+; RUN: llvm-spirv -r %t.spv -o %t.rev.bc
```

Comment:
Since this test checks translation in both ways, it should be moved into `transcoding` sub-dir

Reply:
I've added two-way translation tests for all cases from specification: https://www.khronos.org/registry/OpenCL/extensions/intel/cl_intel_media_block_io.txt